cmd/internal/shared/identityserver/config.go

```diff
@@ -41,6 +41,7 @@ var DefaultIdentityServerConfig = identityserver.Config{
 				JSFiles:       []string{"account.js"},
 			},
 			FrontendConfig: oauth.FrontendConfig{
+				DocumentationBaseURL: "https://thethingsstack.io",
```

Comment:
We haven't used thethingsstack.io in a while, this should be https://thethingsindustries.com/docs.

Reply:
True. This should also be changed in the Console's default config.